jvm/lib/src/test/kotlin/org/odict/DictionaryTest.kt

```diff
@@ -37,7 +37,7 @@ class DictionaryTest {
                 "test.odict")
         val dict = Dictionary("test.odict")
         val entries = dict.lookup("hello")
-
+println(entries)
```

Comment:
Nit: rm console log

Reply:
Gah I thought I caught all of them 😆  Thanks!